src/style.css

```
@@ -1,3 +1,58 @@
 body {
+  --blue-color: #00acdc;
   margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  line-height: 60px;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__item a {
+  display: block;
+  position: relative;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+}
+
+.nav__item a:hover {
+  color: var(--blue-color);
+}
+
+.nav__item a.is-active {
+  color: var(--blue-color);
+}
+
+.nav__item a.is-active:after {
```

1'st point in checklist
```suggestion
.nav__link {
  display: block;
  position: relative;
  text-decoration: none;
  text-transform: uppercase;
  color: black;
}

.nav__link:hover {
  color: var(--blue-color);
}

.is-active {
  color: var(--blue-color);
}

.is-active:after {
```